freud/msd.pyx

```diff
@@ -194,19 +196,21 @@ cdef class MSD:
 
         return self
 
-    @property
+    @Compute._computed_property()
     def box(self):
-        return self.box
+        return self.m_box
```

Comment:
The `box` isn't a computed property for `MSD` - it's a constructor argument, so no need to call `compute` to get the result. This is unusual among `freud` classes, but for a good reason, since the MSD is only meaningful for constant boxes (unlike many other analyses, which can allow box changes across frames). I think this can be reverted to just saving `self.box` in the constructor and having that be directly accessible by the user, as before.